Index.html

```diff
@@ -38,17 +38,209 @@ <h2>
                 a project you need coded, don't hesitate
                 to contact me.
             </p>
-
+         <div class="icon-link">
             <ul>
                 <li><a href="#"><span class="fa fa-twitter"></span></a></li>
                 <li><a href="#"><span class="fa fa-angellist"></span></a></li>
                 <li><a href="#"><span class="fa fa-github"></span></a></li>
                 <li><a href="#"><span class="fa fa-linkedin"></span></a></li>
             </ul>
         </div>
+        </div>
+        </section>
+
+        <section class="Work">
+            
+            <h1>My Recent Work</h1>
```

Comment:
- [ ] As I've mentioned in the summary, kindly remove the white space before the `<h1>` tag.
- [ ] The text is `My Recent Works`, not `My Recent Work`.
- [ ] Your design is missing a tiny green line after the text, as seen in the Figma design.

| Figma Design | Your Design |
| ------------------ | ----------------- |
| ![original](https://user-images.githubusercontent.com/26856491/121514960-853c9a80-c9e4-11eb-8081-7ff1e59232c6.png) | ![clone](https://user-images.githubusercontent.com/26856491/121514973-8a014e80-c9e4-11eb-9ab3-217e651eed29.png) |